src/app/checkout/step-1/step-1.component.js

```diff
@@ -67,6 +69,6 @@ export default angular
     controller: Step1Controller,
     templateUrl: template.name,
     bindings: {
-      changeStep: '&'
```

Comment:
@OzzieOrca I wasn't able to get the function to change the step, so I changed it to bind the variable.  Any downsides to this?